typescript/packages/subsurface-viewer/src/layers/grid3d/grid3dLayer.ts

```diff
@@ -147,10 +152,10 @@ export interface Grid3DLayerProps extends ExtendedLayerProps {
     propertiesData: string | number[] | Float32Array | Uint16Array;
 
     /**
-     * Discrete propety names to be displayed in cursor readouts.
-     * The property values are used as property name indices.
+     * Discrete propety value-name pairs to be displayed in cursor readouts.
+     * The property values are used as the array indices.
      */
-    discretePropertyNames?: string[];
+    discretePropertyValueNames?: IDiscretePropertyValueName[];
```

Comment:
Normally this is a breaking change, but maybe no-one else is using this layer?

Reply:
Only Viz uses the prop as far as I know